objectRecognitionSource/src/main/java/edu/ml/tensorflow/classifier/YOLOClassifier.java

```diff
@@ -46,7 +46,8 @@ public static YOLOClassifier getInstance(float thresh, double[] anchorArray, int
                 anchors = anchorArray;
             }
         }
-
+        System.out.println("Grid Size: " + gridSize);
+        System.out.println("Frame Size: " + frameSize);
```

Comment:
change to log.debug or remove, please

Reply:
My mistake, this was for my personal debugging. Removing these.